components/EpisodeInfo/EpisodeInfo.tsx

```diff
@@ -1,13 +1,22 @@
 import styles from "./EpisodeInfo.module.css";
+import { useRouter } from "next/router";
 
 export type EpisodeProps = {
   episodeName: string;
   imgSrc: string;
 };
 
 function EpisodeInfo({ episodeName, imgSrc }: EpisodeProps) {
+  const router = useRouter();
   return (
     <div className={styles.info_container}>
+      <img
```

Comment:
If this is only used for navigation, better use a `<Link>`/`<a>` here. 
https://nextjs.org/docs/api-reference/next/link

Same issue in `ReferenceList`